lib/geordi/cucumber.rb

```diff
@@ -4,69 +4,33 @@
 # This require-style is to prevent Ruby from loading files of a different
 # version of Geordi.
 require File.expand_path('interaction', __dir__)
-require File.expand_path('firefox_for_selenium', __dir__)
 require File.expand_path('settings', __dir__)
 
 module Geordi
   class Cucumber
 
-    VNC_DISPLAY = ':17'.freeze
-    VNC_PASSWORD_FILE = File.expand_path('~/.vnc/passwd').freeze # default for "vncpasswd"
-    VNC_SERVER_DEFAULT_OPTIONS = "-localhost -nolisten tcp -geometry 1280x1024 -rfbauth #{VNC_PASSWORD_FILE}".freeze
-    VNC_SERVER_COMMAND = "vncserver #{VNC_DISPLAY} #{ENV.fetch('GEORDI_VNC_OPTIONS', VNC_SERVER_DEFAULT_OPTIONS)}".freeze
-    VNC_VIEWER_COMMAND = "vncviewer -passwd #{VNC_PASSWORD_FILE}".freeze
-    VNC_ENV_VARIABLES = %w[DISPLAY BROWSER LAUNCHY_BROWSER].freeze
+    ENV_VARIABLES = %w[DISPLAY BROWSER LAUNCHY_BROWSER].freeze
 
     def run(files, cucumber_options, options = {})
       self.argv = files + cucumber_options.map { |option| option.split('=') }.flatten
       self.settings = Geordi::Settings.new
 
       consolidate_rerun_txt_files
       show_features_to_run
-      setup_vnc if settings.use_vnc?
 
       command = use_parallel_tests?(options) ? parallel_execution_command : serial_execution_command
       Interaction.note_cmd(command) if options[:verbose]
 
       puts # Make newline
-      system command # Util.run! would reset the Firefox PATH
-    end
-
-    def launch_vnc_viewer(source = VNC_DISPLAY)
-      fork do
-        error = capture_stderr do
-          system("#{VNC_VIEWER_COMMAND} #{source}")
-        end
-        unless $?.success?
-          if $?.exitstatus == 127
-            Interaction.fail 'VNC viewer not found. Install it with `geordi vnc --setup`.'
-          else
-            Interaction.note 'VNC viewer could not be opened:'
-            puts error
-            puts
-          end
-        end
-      end
+      system command
     end
 
     def restore_env
-      VNC_ENV_VARIABLES.each do |variable|
+      ENV_VARIABLES.each do |variable|
         ENV[variable] = ENV["OUTER_#{variable}"]
       end
     end
```

Comment:
This must be removed, as you're not setting the OUTER_* vars any more. Please do an extra check for similar usage.